conf/airframes/ENAC/quadrotor/maya_indoor.xml

```diff
@@ -0,0 +1,219 @@
+    <!-- reference acceleration for attitude control -->
+    <define name="REF_ERR_P" value="120"/>
+    <define name="REF_ERR_Q" value="130"/>
+    <define name="REF_ERR_R" value="120"/>
+    <define name="REF_RATE_P" value="15"/>
+    <define name="REF_RATE_Q" value="15"/>
+    <define name="REF_RATE_R" value="10"/>
+    <define name="MAX_R" value="60" unit="deg/s"/>
+
+    <!-- second order filter parameters -->
+    <define name="FILT_CUTOFF" value="4.0"/>
+    <define name="FILT_CUTOFF_R" value="4.0"/>
+
+    <!-- Adaptive Learning Rate -->
+    <define name="USE_ADAPTIVE" value="FALSE"/>
+    <define name="ADAPTIVE_MU" value="0.0001"/>
+
+    <!-- Full INDI -->
+    <!-- control effectiveness -->
+    <define name="G1_ROLL"   value="{  -20.0,  -20.0,  20.0,  20.0 }"/>
+    <define name="G1_PITCH"  value="{   20.0,  -20.0, -20.0,  20.0 }"/>
+    <define name="G1_YAW"    value="{   3,    -3,    3,   -3 }"/>
+    <define name="G1_THRUST" value="{  -1., -1., -1., -1.}"/>
+    <!--Counter torque effect of spinning up a rotor-->
+    <define name="G2"        value="{80, -80, 80, -80.0 }"/>
+    <!-- first order actuator dynamics -->
+    <define name="ACT_FREQ" value="{10.10, 10.10, 10.10, 10.10}"/>
+    <define name="ESTIMATION_FILT_CUTOFF" value="4.0"/>
+    <!--Priority for each axis (roll, pitch, yaw and thrust)-->
+    <define name="WLS_PRIORITIES" value="{1000, 1000, 1, 100}"/>
+  </section>
+
+  <section name="GUIDANCE_V" prefix="GUIDANCE_V_">
+    <define name="REF_MIN_ZDD" value="-0.4*9.81"/>
+    <define name="REF_MAX_ZDD" value=" 0.4*9.81"/>
+    <define name="REF_MIN_ZD" value="-2."/>
+    <define name="REF_MAX_ZD" value=" 2."/>
+    <define name="HOVER_KP" value="90"/>
+    <define name="HOVER_KD" value="110"/>
+    <define name="HOVER_KI" value="10"/>
+    <define name="NOMINAL_HOVER_THROTTLE" value="0.35"/>
+    <define name="ADAPT_THROTTLE_ENABLED" value="TRUE"/>
+    <define name="ADAPT_NOISE_FACTOR" value="1."/>
+    <define name="ADAPT_INITIAL_HOVER_THROTTLE" value="0.3"/>
+  </section>
+
+  <section name="GUIDANCE_H" prefix="GUIDANCE_H_">
+    <define name="MAX_BANK" value="30" unit="deg"/>
+    <define name="PGAIN" value="41"/>
+    <define name="DGAIN" value="108"/>
+    <define name="IGAIN" value="20"/>
+    <define name="NGAIN" value="0"/>
+    <!-- feedforward -->
+    <define name="AGAIN" value="0"/>
+    <define name="REF_MAX_SPEED" value="0.5"/>
+    <define name="REF_MAX_ACCEL" value="2."/>
+  </section>
+
+  <section name="GUIDANCE_INDI" prefix="GUIDANCE_INDI_">
+    <define name="SPECIFIC_FORCE_GAIN" value="-500.0"/>
+    <define name="THRUST_DYNAMICS_FREQ" value="25.64"/>
+    <define name="RC_DEBUG" value="FALSE"/>
+    <define name="SPEED_GAINZ" value="2.5"/>
+  </section>
+
+  <section name="NAV">
+    <define name="ARRIVED_AT_WAYPOINT" value="0.25"/>
+    <define name="NAV_CLIMB_VSPEED" value="0.8"/>
+    <define name="NAV_DESCEND_VSPEED" value="-0.5"/>
+    <define name="NAV_CARROT_DIST" value="18"/>
```

Comment:
Should we remove this ?

Reply:
not needed but makes no real difference in practice